pyfr/solvers/aceuler/elements.py

```diff
@@ -62,24 +62,28 @@ def set_backend(self, *args, **kwargs):
             self.kernels['tdisf_curved'] = lambda uin: self._be.kernel(
                 'tflux', tplargs=tplargs, dims=[self.nupts, r[c]],
                 u=s(self.scal_upts[uin], c), f=s(self._vect_upts, c),
+                extrns={'ac_zeta': 'scalar fpdtype_t'},
```

Comment:
Could you set this as a variable eg `exargs` and then pass that to each kernel? Should reduce the chance of any errors.